css/css-variables/variable-presentation-attribute.html

```diff
@@ -69,7 +69,7 @@
             { property: "font-family", valuesToTest:["Arial", "Times New Roman"], default: "Times New Roman" },
             { property: "font-size", valuesToTest:["31px"], default: "16px" },
             { property: "font-size-adjust", valuesToTest:["22", "none"], default: "none" },
-            { property: "font-stretch", valuesToTest:["normal", "ultra-condensed", "extra-condensed", "condensed", "semi-condensed", "semi-expanded", "expanded", "extra-expanded", "ultra-expanded"], default: "normal" },
```

Comment:
Couldn't these test arguments stay as keywords and only the expectations to be converted to percentages?

Reply:
Maybe.

That would be a significant change to the structure of this test, which doesn’t do that for any other property (setting with a different syntax than expected).

But I think a test like that would be great too.

Reply:
Seems fair to leave these as-is and then add the extra test